src/Routing/Filter/LocaleSelectorFilter.php

```diff
@@ -17,6 +17,7 @@
 use Cake\Event\Event;
 use Cake\Routing\DispatcherFilter;
 use Locale;
+use Cake\I18n\I18n;
```

Comment:
There is a small problem regarding coding standards here. This list of classes needs to be alphabetically sorted